### DISCUSSION TITLE
Pre-fill RSVP with family member details

### DISCUSSION THREAD

**Discussion by Sp0run** [liedekef/events-made-easy#66] [General]
I have just set up a family membership for my club and currently I used the logged in details to pre-fill RSVP form values and save the user time. I presume that this will carry on in much the same way even if the user has family members (ie it will take the logged in users details). But if the logged in user wanted to sign up a family member (who doesnt have a WP sign in) for an event is there currently a way to pre-fill RSVP details for them?

For example, I'm a member of another club with my son, when i sign up for an event I can use a drop down to select myself or my son for the event.

Is there anything similar I can do in EME?

**Comment by liedekef** [comment]
While it would be possible (the info is there), it is currently not possible in EME.
